extras/logger-postgres/Containerfile

```diff
@@ -0,0 +1,15 @@
+ARG LOGGER_POSTGRES_TAG=
+
+FROM docker.io/postgres:${LOGGER_POSTGRES_TAG}
+
+RUN apt update && apt install -y curl python3 python3-aiohttp
+
+COPY watch /usr/sbin/watch
```

Comment:
I don't think we need the watches and all for local development

Reply:
Cool, I'll remove it.

Reply:
I think we can remove the logger-postgres from the makefile as well, right? And just have it in the docker-compose.dev.yml

Reply:
Yes, we can remove it from the Makefile too :smile: